Gopkg.toml

```diff
@@ -31,6 +39,21 @@ required = [
   name = "k8s.io/code-generator"
   version = "kubernetes-1.13.4"
 
+[[override]]
```

Comment:
Same - not used anywhere

Reply:
See above, I had the same issue here as well.